morpho-checkpoint-node/src/Morpho/RPC/Abstract.hs

```diff
@@ -46,8 +45,20 @@ data RpcTrace ev i o
   = RpcTrace (RpcMethod i o) i (RpcEvent ev o)
 
 instance Show ev => Show (RpcTrace ev i o) where
-  show (RpcTrace method@GetLatestBlock request event) = "RpcTrace " <> Prelude.show method <> " " <> Prelude.show request <> " " <> Prelude.show event
-  show (RpcTrace method@PushCheckpoint request event) = "RpcTrace " <> Prelude.show method <> " " <> Prelude.show request <> " " <> Prelude.show event
+  show (RpcTrace method@GetLatestBlock request event) =
```

Comment:
I don't think we should need any `Show` instances anymore, so this should be removed